iradix.go

```diff
@@ -183,6 +183,28 @@ func (t *Txn) writeNode(n *Node, forLeafUpdate bool) *Node {
 	return nc
 }
 
+// Visit all the nodes in the tree under n, and add their mutateChannels to the transaction
+// Returns the size of the subtree visited
+func (t *Txn) recursiveTrackChannelAndCount(n *Node) int {
```

Comment:
I like this form - I'd rename this to `trackChannelsAndCount` to have a little simpler name.